baseline.yaml

```
@@ -649,6 +649,28 @@ criteria:
     security_insights_value: # TODO
     scorecard_probe: # TODO
 
+      - id: OSPS-DO-19   
+    maturity_level: 3
+    category: Documentation
+    criteria: |
+       The project MUST perform an external security
+       review or audit to understand the most likely 
+       and impactful problems that could occur within the 
+       software, and make plans to address those
+       problems.
+    objective: |
+       Projects need to have a formally documented 
+       exteranl security audit/review/assessment and 
+       provide evidence on request.
+    implementation: |
+       Create a status check that checks the project's
+       version control system for evidence or statements
+       that demostrate the project has been reviewed by 
+       an external security professional.
```

Can we link to e.g. `security-artifacts.other-artifacts` from `SECURITY-INSIGHTS.yaml`, with a well-known `artifact-name` field, e.g. `"security-audit-results"`?  (Older audits could be named as `"security-audit-results-$DATE"` if needed, though it's not clear that artifact names need to be unique in security insights.)

```suggestion
       Create a status check that checks the project's
       version control system for evidence or statements
       that demostrate the project has been reviewed by 
       an external security professional.
       
       This information could be recorded in [`security-artifacts.other-artifacts`
       from `SECURITY-INSIGHTS.yaml`](https://github.com/ossf/security-insights-spec/blob/main/specification/security-artifacts.md),
       with a well-known `artifact-name` field of `"security-audit-results"`.
```